addons/website/views/snippets/s_images_constellation.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_images_constellation" name="Images constellation">
+    <section class="s_images_constellation o_cc o_cc5 pt48 pb48">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="23">
+                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-4 col-lg-4 o_snippet_mobile_invisible d-none d-lg-block" style="z-index: 1; grid-area: 3 / 6 / 7 / 9; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px;">
```

Comment:
Even if you don't plan to let user use col, i'd correct `col-lg-*` to match the `g-col-lg-*` for all grid items;